package.json

```diff
@@ -67,7 +67,7 @@
   },
   "dependencies": {
     "@sendbird/chat": "^4.10.1",
-    "@sendbird/uikit-tools": "0.0.1-alpha.40",
+    "@sendbird/uikit-tools": "0.0.1-alpha.42",
```

Comment:
This new version support the newly added option: typingIndicatorTypes

Reply:
Could you update to `0.0.1-alpha.43`?
there are fixes about the file extensions (related about https://github.com/sendbird/sendbird-uikit-core-ts/pull/55 issue)

Reply:
done

Reply:
Could you run the yarn install command? It seems CI is broken in the installation step.

Reply:
oh yes done. thank you it was the new package depency version update